cfgmgr/vrfmgr.cpp

```diff
@@ -235,8 +308,19 @@ void VrfMgr::doTask(Consumer &consumer)
                         it = consumer.m_toSync.erase(it);
                         continue;
                     }
+                    vector<FieldValueTuple> vrfFvVector;
+                    // Incase of MGMT_VRF_CONFIG table event, the fields mgmtVrfEnabled, 
+                    // in_band_mgmt_enabled should be ignored.
+                    if (consumer.getTableName() != CFG_MGMT_VRF_CONFIG_TABLE_NAME) 
```

Comment:
Is this really required. I think it should be ok to leave the fields populated.

Reply:
ok, we can populate those fields but it won't be of any use for now.